### DISCUSSION TITLE
Allow map-zoom on car overview

### DISCUSSION THREAD

**Discussion by leewillis77** [teslamate-org/teslamate#921] [Ideas]
**What would you like to be added**:
Currently the overview of a car in Teslamate (http://teslamate.server.address:4000/) shows a small square map, and a summary of key information. If the car is driving, the map auto-updates and follows the car on its journey. Unfortunately, the zoom of that map is fixed, and often the small size, and fixed zoom mean there's not enough context for the map to actually be useful.

For example, the current map view I'm looking for my car looks like this:
![Screenshot 2020-09-14 at 15 57 56](https://user-images.githubusercontent.com/1097338/93102730-a9f92100-f6a3-11ea-9b1b-f3118cd0177a.png)

If I adjust the zoom I could get enough context about the position for this to be much more useful.

**Comment by m1001101** [comment]
Same request, thanks ;)

**Comment by andyross** [comment]
Also a link to the current grafana drive details dashboard (which has a very nicely browsable interface) would be very helpful.

**Comment by bogosj** [comment]
It appears to be controlled here: https://github.com/adriankumpf/teslamate/blob/master/assets/js/hooks.js#L165

I don't see any way to configure this, but setting this to `true` puts a nice +/- control on the top left of the map.

@adriankumpf - do you have opinions on how this could be implemented? I'm happy to help.

**Comment by leewillis77** [comment]
I *think* you'd need to add `data-zoom="1"` to the map container here:
https://github.com/adriankumpf/teslamate/blob/master/lib/teslamate_web/live/car_live/summary.html.heex#L3

Then, you'd also need to update the calls to `setView()` to send the current zoom level rather than just 17 at:
https://github.com/adriankumpf/teslamate/blob/master/assets/js/hooks.js#L182
and 
https://github.com/adriankumpf/teslamate/blob/master/assets/js/hooks.js#L190

Unfortunately I don't have a local build right now to try this out on, maybe someone else can see if that works...

**Comment by bogosj** [comment]
https://github.com/adriankumpf/teslamate/pull/2184 works for me.

**Comment by bogosj** [comment]
PR merged @leewillis77

**Reply by leewillis77** [reply]
Thanks @bogosj @adriankumpf
